cafebabel/templates/_social-networks-macro.html

```diff
@@ -0,0 +1,8 @@
+{% macro social_networks() %}
+  <ul>
+    <li><a href={{ social_network_url_for('facebook') }} class=icon-facebook></a></li>
+    <li><a href={{ social_network_url_for('twitter') }} class=icon-twitter></a></li>
+    <li><a href={{ social_network_url_for('instagram') }} class=icon-instagram></a></li>
+    <li><a href={{ social_network_url_for('youtube') }} class=icon-youtube></a></li>
+  </ul>
+{% endmacro %}
```

Comment:
Here we want to have linkedin but only in the footer. Do you think it is best to add it to this `macro`, then hide it on the right top corner? OR don't use this `macro` for the footer?

Reply:
Mmmh, tricky.

Reply:
Why not just writing `<a href={{ social_network_url_for('youtube') }} class=icon-youtube></a>` in the footer?

Reply:
@cafebabel-webmaster I pushed something for you to get the logic, can you adapt (especially the linkedin icon)?

Reply:
ha yes ! `with_linkedin=False` 👍